matlab/schemas/+experiment/Projector.m

```diff
@@ -0,0 +1,15 @@
+%{
+experiment.Projector (lookup) # 
+projector_id        : tinyint                       # projector id
+---
+pixel_width         : smallint                      # number of pixels in width
+pixel_height        : smallint                      # number of pixels in height
+red                 : enum('UV', 'Green', 'None')   # color to be used for red channel
```

Comment:
this enum does not match the enum in .py

Reply:
fixed